Header header
float64 target_velocity           # target velocity, in m/s
float64 target_steering_angle     # target steering angle, in degrees
float64 cross_track_error         # perpendicular distance to intended trajectory, in meters
float64 heading_error             # heading error, in degrees
float64 timestamp 

